app/design/adminhtml/default/default/template/downloadable/product/edit/downloadable/links.phtml

```diff
@@ -17,7 +17,8 @@
 
 /**
  * @see Mage_Downloadable_Block_Adminhtml_Catalog_Product_Edit_Tab_Downloadable_Links
- * @var $this Mage_Downloadable_Block_Adminhtml_Catalog_Product_Edit_Tab_Downloadable_Links
+ * @var Mage_Downloadable_Block_Adminhtml_Catalog_Product_Edit_Tab_Downloadable_Links $this
+ * @var Mage_Downloadable_Block_Adminhtml_Catalog_Product_Edit_Tab_Downloadable_Links $this
```

Comment:
The line seems to be doubled. If it is an error there are more places for changes.